src/sedpack/io/dataset.py

```diff
@@ -356,6 +356,7 @@ def read_and_decode(self, tf_dataset: TFDatasetT,
     def as_tfdataset(  # pylint: disable=too-many-arguments
```

Comment:
Can we remove these pylint directive now or is it still triggering?

Reply:
Different error (still triggering).

Reply:
Ok